doc/dev/osd_internals/scrub.rst

```diff
@@ -28,3 +31,11 @@ State variables
 - Initiated scrub state is  must_scrub && !must_deep_scrub && !time_for_deep
 - Initiated scrub after osd_deep_scrub_interval state is must scrub && !must_deep_scrub && time_for_deep
 - Initiated deep scrub state is  must_scrub && must_deep_scrub
+
+Scrub Reservations
+------------------
+
+An OSD daemon command dumps total local and remote reserations::
+
+  ceph daemon osd.<id> dump_scrub_reservations
+
```

Comment:
might want to add something similar for `ceph daemon osd.<id> dump_recovery_reservations` in dev/osd_internals/recovery_reservation.rst